Administrator Guide/Managing Volumes.md

```diff
@@ -260,38 +261,235 @@ set).
+
+Steps to remove brick Server1:/home/gfs/r2_1 and add Server1:/home/gfs/r2_2:
+
+1. Here is the initial volume configuration:
+```
+    Volume Name: r2
+    Type: Distribute
+    Volume ID: 25b4e313-7b36-445d-b524-c3daebb91188
+    Status: Started
+    Number of Bricks: 2
+    Transport-type: tcp
+    Bricks:
+    Brick1: Server1:/home/gfs/r2_0
+    Brick2: Server1:/home/gfs/r2_1
+```
+
+2. Here are the files that are present on the mount:
+
+```
+    ⚡ ls
+    1  10  2  3  4  5  6  7  8  9
+```
+3. Add the new brick - Server1:/home/gfs/r2_2 now:
+
+```
+    ⚡ gluster volume add-brick r2 Server1:/home/gfs/r2_2
+    volume add-brick: success
+```
+4. Start remove-brick using the following command:
+
+```
+    ⚡ gluster volume remove-brick r2 Server1:/home/gfs/r2_1 start
+    volume remove-brick start: success
+    ID: fba0a488-21a4-42b7-8a41-b27ebaa8e5f4
+```
+
+5. Wait until remove-brick is completed. Once remove-brick is completed it
+shows the following output:
+
+```
+    ⚡ gluster volume remove-brick r2 Server1:/home/gfs/r2_1 status
+                                        Node Rebalanced-files          size       scanned      failures       skipped               status   run time in secs
+                                   ---------      -----------   -----------   -----------   -----------   -----------         ------------     --------------
+                                   localhost                5       20Bytes            15             0             0            completed               0.00
+```
+
+7. Now we can safely remove the old brick, so commit the changes:
+
+```
+    ⚡ gluster volume remove-brick r2 Server1:/home/gfs/r2_1 commit
+    Removing brick(s) can result in data loss. Do you want to Continue? (y/n) y
+    volume remove-brick commit: success
+```
+
+6. Here is how the volume configuration will be after the remove-brick.
+
+```
+    Volume Name: r2
+    Type: Distribute
+    Volume ID: 25b4e313-7b36-445d-b524-c3daebb91188
+    Status: Started
+    Number of Bricks: 2
+    Transport-type: tcp
+    Bricks:
+    Brick1: Server1:/home/gfs/r2_0
+    Brick2: Server1:/home/gfs/r2_2
+```
+
+7. Check the contents of the mount:
+
+```
+    ⚡ ls
+    1  10  2  3  4  5  6  7  8  9
+```
+
+**Replacing brick in Replicate/Distributed Replicate volumes**
+
+This section of the document contains how brick: `Server1:/home/gfs/r2_0` is replaced with brick: `Server1:/home/gfs/r2_5` in volume `r2` with replica count `2`.
+
+```
+    Volume Name: r2
+    Type: Distributed-Replicate
+    Volume ID: 24a0437a-daa0-4044-8acf-7aa82efd76fd
+    Status: Started
+    Number of Bricks: 2 x 2 = 4
+    Transport-type: tcp
+    Bricks:
+    Brick1: Server1:/home/gfs/r2_0
+    Brick2: Server2:/home/gfs/r2_1
+    Brick3: Server1:/home/gfs/r2_2
+    Brick4: Server2:/home/gfs/r2_3
+```
+
+
+Steps:
+0. Make sure there is no data in the new brick Server1:/home/gfs/r2_5
```

Comment:
Same comment here. These numbered lists are not rendered properly for some reason.